src/ngx_http_lua_timer.c

```diff
@@ -579,6 +589,9 @@ ngx_http_lua_timer_handler(ngx_event_t *ev)
 
     r = ngx_http_lua_create_fake_request(c);
     if (r == NULL) {
+        ngx_log_error(NGX_LOG_ALERT, ngx_cycle->log, 0,
+                      "failed to create fake request to run timer (co: %p)",
+                      tctx.co);
```

Comment:
Hmm, can we simply put this under the `failed` code label so that we do not have to repeat ourselves?

Reply:
@agentzh 
The error messages before `goto failed` are different. They are in `failed to do something to run timer` format, but the content in `do something` part is not repeated.

Reply:
@agentzh 
Not all the error messages before `failed` are in the same format, like this one:
https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_timer.c#L575

Reply:
@agentzh Ping. This seems fine as is?

Reply:
I don't have strong opinion on this one. It's fine as is.